internal/common/common.go

```diff
@@ -43,6 +43,8 @@ const (
 	X86CPUArchitecture     = "x86_64"
 	DefaultCPUArchitecture = X86CPUArchitecture
 	ARM64CPUArchitecture   = "arm64"
```

Comment:
do we really need to have (and check) for both names?

I mean, we do have also amd64 and x86_64 as both names for x86, but they're used in different places. I'd think that it would be the same for arm ...

:?

Reply:
what Javi said

Reply:
arm is used for release image and changing it to aarch will be pretty painful right now
Though agent will always return aarch